### ISSUE TITLE
language (nl_NL) not loading correct for popup login and woopopup extension

### ISSUE THREAD

**Issue by codeagencybe** [oceanwp/oceanwp#84]
Hello Nicolas,

I'm using the woopopup and login popup extension in a project, but it's not loading translations correctly. 

The login popup, it shows party English and partly Dutch. Can you explain how to get everything in Dutch? 

I opened the .po file with poedit for the popup login, but it shows strings from the woopopup extension... how is that possible? 

Also the woopopup is always showing the shortcode in English for bottom text -> [oceanwp_woo_free_shipping_left]
How and where to translate that part please? 

Thanks!

**Comment by codeagencybe** [comment]
You can see screenshot here for the mismatch strings in .po file

![schermafbeelding 2018-05-02 om 13 25 50](https://user-images.githubusercontent.com/7568546/39521103-6d4b136c-4e0d-11e8-8d30-54da60d9a233.png)

**Comment by nicolas-owp** [comment]
Hello, there is no Dutch translation for the popup login, you need to open the pot file, in poedit, click the Updates link, then translate it and save like this "ocean-popup-login-de_DE".
For the shortcode it is via Ocean Extra.
Tell me if it is good.

**Comment by codeagencybe** [comment]
Hello Nicolas,

yes I think I found the culprit... strangely the languages files I got with the login popup extension was the same as the woo popup. 
I have re-downloaded everything and uploaded manually, now it's fine. 
The Dutch translations I have done myself. 
I just found it odd why the theme itself does has Dutch translations but the extensions not... 

Anyway, I got it sorted out now. All good now. 
Thanks!
